api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1436,9 +1436,9 @@ public List<Concept> getPossibleValues() {
 	 * @see org.openmrs.Attributable#hydrate(java.lang.String)
 	 */
 	@Override
-	public Concept hydrate(String s) {
+	public Concept hydrate(String reference) {
 		try {
-			return Context.getConceptService().getConcept(Integer.valueOf(s));
```

Comment:
@wikumChamith is this supposed to be part of this pull request?

Reply:
Where is this from :sweat_smile: